common.rkt

```diff
@@ -175,10 +175,10 @@
     ((not v) 1)
     ((eqv? u #t) -1)
     ((eqv? v #t) 1)
-    ((number? u) (if (number? v) (if (< u v) -1 1) -1))
-    ((number? v) 1)
     ((symbol? u) (if (symbol? v) (if (symbol<? u v) -1 1) -1))
     ((symbol? v) 1)
+    ((number? u) (if (number? v) (if (< u v) -1 1) -1))
+    ((number? v) 1)
```

Comment:
This is fine, though if you're trying to keep variables on the left of disequalities, this won't help with `()` or booleans.  You can't reliably fix that issue without preserving variables as an independent type, and treat them specially in `pretty-diseq`.